src/Camelot.ViewModels.Windows/ShellIcons/WindowsShellIconsService.cs

```diff
@@ -23,19 +23,14 @@ public class WindowsShellIconsService : IShellIconsService
 
     public ImageModel GetIconForExtension(string extension)
     {
-        if (string.IsNullOrEmpty(extension))
+        if (!_pathService.IsExtension(extension))
```

Comment:
Let's remove this check and the corresponding method in `IPathService`. Project standard is to avoid redundant checks especially given that extension is following the same standards across the codebase. Also `IsExtension` doesn't have too much value